src/vm/methodtablebuilder.cpp

```diff
@@ -10257,6 +10278,16 @@ void MethodTableBuilder::CheckForSystemTypes()
         {
             pMT->SetIsNullable();
         }
+#ifdef FEATURE_SPAN_OF_T
+        else if (strcmp(name, g_SpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
+        else if (strcmp(name, g_ReadOnlySpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
```

Comment:
This only runs for types in corelib. The callsite of this method is under `if (GetModule()->IsSystem())`.